### ISSUE TITLE
Inform the user when their input action will open a new window

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-admintools#161]
A user should be warned when interacting with an element is opening a new window, as stated in [WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/general/G201)
